addons/mail/static/src/models/rtc/rtc.js

```diff
@@ -1278,6 +1279,13 @@ registerModel({
         recoveryDelay: attr({
             default: 3000,
         }),
+        /**
+         * Object { token: peerConnection<RTCPeerConnection> }
+         * Contains the RTCPeerConnection established with the other rtc sessions.
+         * Exposing this field and keeping references to closed peer connections may lead
+         * to difficulties reconnecting to the same peer.
```

Comment:
According to this comment, the "main model" that we should use might be better `RtcSession` (which already exists) rather than peer connection? Session is the main holder of the "token", and all new models should be related to it in some way?